scss/utilities/_overflow.scss

```diff
@@ -0,0 +1,16 @@
+// stylelint-disable scss/dollar-variable-default
+$directions: x, y;
+$name-map: (
+  hide: hidden,
+  auto: auto,
+  scroll: scroll,
+  visible: visible,
+);
+
+@each $name, $value in $name-map {
+  .overflow-#{$name} { overflow: $value; }
+
+  @each $direction in $directions {
+    .overflow-#{$direction}-#{$name} { overflow-#{$direction}: $value; }
```

Comment:
Wondering if we start with just the standard overflow instead of directional options? Easier to add than it is to remove :).

Reply:
Huh? So you're thinking .overflow-hidden, .overflow-auto, .overflow-visible, then do the x and y versions?